### PR TITLE
Hydroponic plants now react to radiation!

### PR DESCRIPTION
**Pull request by DeityLink**
image

:cl:
* rscadd: Plants exposed to radiation now have a chance to mutate. Young plants planted in the past minute have a high chance of mutating, while older ones have a very small chance of a minor gene mutation (if all your trays in Hydroponics have plants and a radiation storm occurs, expect 3-5 of them to display a small mutation). Furthermore, closing the tray's lid completely removes any chance of radiation-induced mutations.
* tweak: If a plant mutates to different species, it will no longer have an additional small mutation applied at the same time.

### CODE REVIEW

**Comment by SECBATON-GRIFFON** [comment]
mutating plants with the shard looks so dumb and fun and risk-reward all at the same time, i love it
also you know, its more department interaction

**Review by west3436 — APPROVED** [review]

**Comment by SonixApache** [comment]
radiation, not radiation*s*

File: code/modules/power/singularity/collector.dm [hunk 1]
```diff
@@ -134,13 +134,18 @@ var/global/list/rad_collectors = list()
 	else
 		update_icons()
 
+//This completely ignores Z Levels, so you can collect radiations from the station's singulo from the asteroid.
```
Comment (SonixApache) [inline code/modules/power/singularity/collector.dm:137]: ☢️🥖☢️

**Comment by Killette2** [comment]
I will not be responsible when regular death nettle turns into kudzu when the radiation RNG wills it

**Comment by DeityLink** [comment]
I will be responsible and I hope I'll get the chance to watch such an astronomically low odds event.

You need a "young" weed to get irradiated by a radstorm into a nettle and once more to turn into a death nettle (which can theoretically happen in a single storm), but then you need a non-young death nettle to get irradiated again to get the kudzu gene. 

So for it to happen on its own, you'd need two radstorms in close succession before the death nettle dies of old age.

**Pull request merged**